commit 8879342a67a4c0986dd48f8d4381b94999ada174
Author: Mario Limonciello <mario.limonciello@dell.com>
Date:   Mon Sep 10 13:01:52 2018 -0500

    platform/x86: dell-smbios-wmi: Correct a memory leak
    
    commit affab51082174f60ef71ced8ab5fbe71f00e9ae3 upstream.
    
    ACPI buffers were being allocated but never freed.
    
    Reported-by: Pinzhen Xu <pinzhen.xu@intel.com>
    Signed-off-by: Mario Limonciello <mario.limonciello@dell.com>
    Cc: stable@vger.kernel.org
    Signed-off-by: Darren Hart (VMware) <dvhart@infradead.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/platform/x86/dell-smbios-wmi.c b/drivers/platform/x86/dell-smbios-wmi.c
index fbefedb1c172..548abba2c1e9 100644
--- a/drivers/platform/x86/dell-smbios-wmi.c
+++ b/drivers/platform/x86/dell-smbios-wmi.c
@@ -78,6 +78,7 @@ static int run_smbios_call(struct wmi_device *wdev)
 	dev_dbg(&wdev->dev, "result: [%08x,%08x,%08x,%08x]\n",
 		priv->buf->std.output[0], priv->buf->std.output[1],
 		priv->buf->std.output[2], priv->buf->std.output[3]);
+	kfree(output.pointer);
 
 	return 0;
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1041_8879342a67a4c0986dd48f8d4381b94999ada174.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1041_8879342a67a4c0986dd48f8d4381b94999ada174.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
